### PR TITLE
Janus: install from DEBs

### PR DESCRIPTION
**Pull request by dergraf**
Instead of https://github.com/cargomedia/puppet-packages/pull/999

### CODE REVIEW

**Comment by dergraf** [comment]
@ppp0 please review, cc: @njam, @kris-lab

**Comment by njam** [comment]
Just a note - let's double check this with chris before merging - as it will immediately be applied to production!

**Comment by dergraf** [comment]
sure!

**Comment by ppp0** [comment]
from my POV all looks good, I've re-tested the specs - we had paired woth @dergraf 

still, @kris-lab final check, please

**Comment by ppp0** [comment]
considering #1077, the `Puppetfile` requirement for `bipbip` is ok

File: modules/janus/manifests/plugin/rtpbroadcast.pp [hunk 1]
```diff
@@ -13,6 +13,14 @@
   $job_pattern = 'job-#{md5}'
 ) {
 
+  require 'apt'
```
Comment (ppp0) [inline modules/janus/manifests/plugin/rtpbroadcast.pp]: @dergraf considering https://github.com/cargomedia/debian-packages/issues/66, please remove line 16-23

**Comment by dergraf** [comment]
regarding libsrtp see discussion in https://github.com/cargomedia/debian-packages/issues/66

**Comment by kris-lab** [comment]
Looks good! The only my concern is about `janus::service` and main-config and plugin-config path. Can we do it configurable?

**Comment by dergraf** [comment]
sure, I'll give it a try!

**Comment by ppp0** [comment]
I added some spec tests to visualize... waiting for @dergraf to give me write access to his fork (to push to this PR directly)

**Comment by kris-lab** [comment]
lgtm

**Comment by ppp0** [comment]
will add specs (https://github.com/cargomedia/puppet-packages/pull/1035#issuecomment-165192933) in #1080

**Pull request merged**